birdhouse/components/weaver/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG WEAVER_VERSION=latest
```

Comment:
This is related to https://github.com/bird-house/birdhouse-deploy/pull/114#discussion_r596930805 posted by @dbyrns 

Because Weaver needs the other WPS birds to be *ready* to register them as providers, they must be started first. The problem with docker-compose `depends_on` is that it only ensure the other bird images are *started*, but not necessarily *ready* to receive requests. For instance, `flyingpigeon` is particularly long to boot because of the number of processes it has. 

The `Dockerfile` placed here combines the original Weaver docker (from its repo) with `korbin/dockerize`. 
The added utility allows to ping the birds (`GetCapabilities` requests) until they return 200. Only once those are received, `Weaver` will boot and retrieve the responses from the bird that are now *ready for real*.

The reason I placed this here is that this is very specific to this stack. 
Weaver itself doesn't need this tool and I don't believe it should be up to the source repo to handle external use cases like this. The extra docker-compose is defined such that it will try to build this combined image because it doesn't find it remotely.

Reply:
Ah ok so I am buying "the reason I placed this here is that this is very specific to this stack.".   Can you just pin the version of the dockerize image so this build is reproducible?